### PR TITLE
Delete User functionality completed

### PR DESCRIPTION
**Pull request by upGradShashvat**
User Delete functionality completed

### CODE REVIEW

File: quora-service/src/main/java/com/upgrad/quora/service/business/UserService.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import com.upgrad.quora.service.dao.UserDao;
+import com.upgrad.quora.service.entity.UserEntity;
+import com.upgrad.quora.service.exception.AuthorizationFailedException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+
+@Service
+public class UserService {
```
Comment (shguptamts) [inline quora-service/src/main/java/com/upgrad/quora/service/business/UserService.java]: UserService class not needed .. there is already UserBusinessService present use that class

**Review by shguptamts — CHANGES_REQUESTED** [review]
please do the required changes

**Comment by shguptamts** [comment]
closing this.. as the pull is on wrong branch

File: quora-service/src/main/java/com/upgrad/quora/service/business/UserService.java (hunk 1 above)
Comment (upGradShashvat) [reply quora-service/src/main/java/com/upgrad/quora/service/business/UserService.java]: Merged UserService class into UserBusinessService class. I have commented the code in UserService class, you can delete tne UserService class .. made changed to 3 class -
1. UserService
2. UserBusiessService
3. AdminController

**Pull request merged**